### PR TITLE
Announce Tenzir v4.6

### PR DESCRIPTION
**Pull request by dominiklohmann**
This adds the blog post for Tenzir v4.6, scheduled for Friday, Dec 1st.

### CODE REVIEW

File: web/blog/tenzir-v4.6/tenzir-v4.6-url-expansion.excalidraw.svg
Comment (eliaskosunen) [inline web/blog/tenzir-v4.6/tenzir-v4.6-url-expansion.excalidraw.svg:1]: `| decompress zstd` -> `| compress zstd`

File: web/blog/tenzir-v4.6/index.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+```
+from tcp://localhost:514 read syslog
+| parse content --into content cef
+```
+
+Here's an example input and output:
+
+```syslog {0} title="Input"
+Nov 13 16:59:59 host123 FOO: CEF:0|FORCEPOINT|Firewall|6.6.1|0|Generic|0|deviceExternalId=Master FW node 1 dvc=10.1.1.40 dvchost=10.1.1.40 msg=log server connection established deviceFacility=Logging System rt=Jan 17 2020 08:52:10
```
Comment (eliaskosunen) [inline web/blog/tenzir-v4.6/index.md:90]: I made a note about this here: https://github.com/tenzir/tenzir/pull/3665#discussion_r1410669284

File: web/blog/tenzir-v4.6/index.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+another format.
+
+Onboarding custom data sources is a pain for every SOC operations team. We've
+seen CSV where some columns are NDJSON, CEF in Syslog, and grok patterns in CEF
+in Syslog. With the `parse` operator, this no longer has to be as painful, and
+you can finally spend more time working with your data than onboarding it.
+
+## Rapid Prototyping with the Python Operator
+
+The `python` operator allows for modying events using Python. Here are soom cool
```
Comment (eliaskosunen) [inline web/blog/tenzir-v4.6/index.md:155]:
```suggestion
The `python` operator allows for modying events using Python. Here are some cool
```

File: web/blog/tenzir-v4.6/index.md (hunk 1 above)
Comment (dominiklohmann) [reply web/blog/tenzir-v4.6/index.md:90]: I discussed this briefly with Jannis in a call just now, and we both agree that this is a wontfix for the current release and that we will fix it for a later one.

File: web/blog/tenzir-v4.6/tenzir-v4.6-url-expansion.excalidraw.svg
Comment (dominiklohmann) [reply web/blog/tenzir-v4.6/tenzir-v4.6-url-expansion.excalidraw.svg:1]: Fixed in 805f1c0217

File: web/blog/tenzir-v4.6/index.md (hunk 2 above)
Comment (dominiklohmann) [reply web/blog/tenzir-v4.6/index.md]: Fixed in 805f1c0217, thanks!

**Review by lava — APPROVED** [review]

**Pull request merged**